app/assets/stylesheets/modern-business.scss

```diff
@@ -239,6 +239,20 @@ footer {
   margin-right: 550px;
 }
 
+@media(max-width:767px) {
+    #row-donate-form {
```

Comment:
Avoid using id selectors<br>Line should be indented 2 spaces, but was indented 4 spaces